File: PRAK404-2210817210023-Muhammad Shofy Akmal.py
pil = 1
while pil == 1 :
    print("Pilih program\n1. Penjumlahan\n2. Pengurangan\n3. Perkalian\n4. Pembagian\n5. Exit")
    print("Masukkan Pilihan : ", end=''); a = int(input())
    if a<1 or a>1 : 
        print("Input anda salah, silahkan coba lagi\n")
    elif a==5 : 
        print("\nTerimakasih, telah menggunakan kalkulator Kholid :)\n"); break
    else : 
        print("Masukkan Nilai Pertama : ", end=''); b=float(input())
        print("Masukkan Nilai Kedua   : ", end=''); c=float(input())
        if a == 1 : print("Hasil Penjumlahan antara %.2f dengan %.2f adalah %.2f\n" % (b,c,b+c))
        elif a == 2 : print("Hasil Pengurangan antara %.2f dengan %.2f adalah %.2f\n" % (b,c,b-c))
        elif a == 3 : print("Hasil Perkalian antara %.2f dengan %.2f adalah %.2f\n" % (b,c,b*c))
        else : print("Hasil Pembagian antara %.2f dengan %.2f adalah %.2f\n" % (b,c,b/c))

File: PRAK404-2210817210023-MuhammadShofyAkmal.c
#include <stdio.h>
int main () {
float a,b,c;
while (1){ 
printf ("Pilih program\n1. Penjumlahan\n2. Pengurangan\n3. Perkalian\n4. Pembagian\n5. Exit\n");
printf ("Masukkan Pilihan  : "); scanf ("%f", &a);
if      (a<1 || a>5)  {printf("Input anda salah, silahkan coba lagi\n");} 
else if (a==5) {printf ("Terimakasih, telah menggunakan kalkulator Akmal");break;}
else {
printf ("Masukkan nilai pertama : "); scanf ("%f", &b);
printf ("Masukkan nilai kedua   : "); scanf ("%f", &c);
if      (a==3) {printf ("Hasil perkalian antara %.2f dengan %.2f adalah %.2f\n", b,c,b*c);}
else if (a==1) {printf ("Hasil penjumlahan antara %.2f dengan %.2f adalah %.2f\n", b,c,b+c);}
else if (a==2) {printf ("Hasil pengurangan antara %.2f dengan %.2f adalah %.2f\n", b,c,b-c);}
else           {printf ("Hasil pembagian antara %.2f dengan %.2f adalah %.2f\n", b,c,b/c);}}}
}